xoa/resource_xenorchestra_vm.go

```diff
@@ -207,7 +207,7 @@ $ xo-cli xo.getAllObjects filter='json:{"id": "cf7b5d7d-3cd5-6b7c-5025-5c935c8cd
   "max": 4,
   "number": 2
 }
-		    
+
 # Updating the VM to use 3 CPUs would happen without stopping/starting the VM
```

Comment:
This is cause the failing `tfplugindocs` run since it removed the same whitespace from the vm resource mardown file.

```
ddelnano@ddelnano-desktop:~/code/terraform-provider-xenorchestra (belfhi/master) $ ~/go/bin/tfplugindocs

ddelnano@ddelnano-desktop:~/code/terraform-provider-xenorchestra (belfhi/master) $ git diff docs/
diff --git a/docs/resources/vm.md b/docs/resources/vm.md
index e979d35..7972c31 100644
--- a/docs/resources/vm.md
+++ b/docs/resources/vm.md
@@ -129,7 +129,7 @@ $ xo-cli xo.getAllObjects filter='json:{"id": "cf7b5d7d-3cd5-6b7c-5025-5c935c8cd
   "max": 4,
   "number": 2
 }
-
+
 # Updating the VM to use 3 CPUs would happen without stopping/starting the VM
 # Updating the VM to use 5 CPUs would stop/start the VM
```

Please run `go generate ./...` from the root of the repo to make those changes locally.